### PR TITLE
Fixed issue #18211: List (Radio) questions columns not in one (ul) tag.

### PR DESCRIPTION
**Pull request by Shnoulle**
Fixed issue #18210: List (Radio) questions columns section not behaving properly
Fixed issue : Multiple choice questions columns not in one (ul) tag. 
Fixed issue : broken HTML for image multiple choice (li without ul)
Dev: theme style solution
Dev: work for limited column number

### CODE REVIEW

**Comment by Shnoulle** [comment]
I start on this one tomorrow : limited number of columns and update theme.css

**Comment by Shnoulle** [comment]
I put major impact because some Question theme can allow multiple column before, and disable after.

File: application/core/QuestionTypes/ListRadio/RenderListRadio.php [hunk 1]
```diff
@@ -58,17 +54,12 @@ public function __construct($aFieldArray, $bRenderDirect = false)
...
-            // First we calculate the width of each column
-            // Max number of column is 12 http://getbootstrap.com/css/#grid
-            $this->iColumnWidth = round(12 / $this->iNbCols);
-            $this->iColumnWidth = ($this->iColumnWidth >= 1) ? $this->iColumnWidth : 1;
-            $this->iColumnWidth = ($this->iColumnWidth <= 12) ? $this->iColumnWidth : 12;
-            $this->iMaxRowsByColumn = ceil($this->getAnswerCount() / $this->iNbCols);
-        } else {
-            $this->iColumnWidth = 12;
-            $this->iMaxRowsByColumn = $this->getAnswerCount() + 3; // No max : anscount + no answer + other + 1 by security
+            /* Adding but didn't work */
```
Comment (Shnoulle) [inline application/core/QuestionTypes/ListRadio/RenderListRadio.php:58]: Must remove …

File: application/views/survey/questions/answer/listradio/columns/column_footer.twig [hunk 2]
```diff
@@ -1,8 +1 @@
-{#
-/**
- * Multiple Choice Html : close column containing item row
- * This view is used only if user set more than one column in the question attribute.
- *
- */
- #}
-</ul>
+{# @deprecated 5.X.Y #}
```
Comment (Shnoulle) [inline application/views/survey/questions/answer/listradio/columns/column_footer.twig:1]: Tu update when we have the real version number

**Comment by Shnoulle** [comment]
Close it : move to dev (potential javascript workaround issue)

**Comment by Shnoulle** [comment]
Not to be merged : keep it for testing.

Wait for BS5 branch …

**Comment by olleharstedt** [comment]
The conflicts need to be cleaned up. @ptelu will have a look too.

**Comment by olleharstedt** [comment]
I'm rebasing this to master branch, if you want it in 6.0.0.

**Comment by Shnoulle** [comment]
> The conflicts need to be cleaned up. @ptelu will have a look too.

Yes, i know … i just set it as develop. Maybe i redo whole (more easy sometime)

**Comment by Shnoulle** [comment]
> I'm rebasing this to master branch, if you want it in 6.0.0.

master is 6.0 ?

**Comment by Shnoulle** [comment]
replace by https://github.com/LimeSurvey/LimeSurvey/pull/3038

**Pull request closed without merging**